Integrations/McAfee-TIE/McAfee-TIE.py

```diff
@@ -49,6 +35,27 @@
 }
 
 
+def validate_certificates_format():
+    if '-----BEGIN PRIVATE KEY-----' not in demisto.params()['private_key']:
+        return_error(
+            "The private key content seems to be incorrect as it doesn't start with -----BEGIN PRIVATE KEY-----")
+    if '-----END PRIVATE KEY-----' not in demisto.params()['private_key']:
+        return_error(
+            "The private key content seems to be incorrect as it doesn't end with -----END PRIVATE KEY-----")
+    if '-----BEGIN CERTIFICATE-----' not in demisto.params()['cert_file']:
+        return_error("The client certificates content seem to be "
+                     "incorrect as they don't start with '-----BEGIN CERTIFICATE-----'")
+    if '-----END CERTIFICATE-----' not in demisto.params()['cert_file']:
+        return_error(
+            "The client certificates content seem to be incorrect as it doesn't end with -----END CERTIFICATE-----")
+    if not demisto.params()['broker_ca_bundle'].lstrip(" ").startswith('-----BEGIN CERTIFICATE-----'):
```

Comment:
Use `not in` as is done above

Reply:
@glicht The reason that I'm not using `not in` is that because this is a list of certificates, if the second certificate starts with -----...--- we'll approve this and continue on. Same for ends with. The lstrip removes all spaces from the begining and is checked in the unittest.
What do you say?

Reply:
I have no problem. But the code is not consistent between tie and dxl integrations. Let's do one thing that works well. See: https://github.com/demisto/content/pull/5440/files#diff-bdb8381bfed45ef70a7592e64adf4e15R130

Reply:
You're 100% right. I must have forgotten to update the TIE function.